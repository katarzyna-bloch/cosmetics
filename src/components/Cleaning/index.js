const Cleaning = () => (     
  <h4>Oczyszczanie to podstawowy krok w dojściu do pięknej skóry</h4>
)

export default Cleaning
