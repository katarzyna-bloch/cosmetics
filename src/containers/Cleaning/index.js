import React, { useEffect } from 'react'
import { connect } from 'react-redux'

import PageWrapper from '../../components/PageWrapper'
import ContentHeader from '../../components/ContentHeader'
import ProductsList from '../../components/ProductsList'
import { requestCleaningProducts } from '../../redux/actions'

export const Cleaning = ({ fetchProducts, products }) => {
  useEffect(() => {
    fetchProducts()
  }, [])

  return (
    <PageWrapper>
      <ContentHeader>Oczyszczanie to podstawowy krok w dojściu do pięknej skóry</ContentHeader>
      <ProductsList
        products={products}
      />
    </PageWrapper> 
  )     
}

const mapStateToProps = ({ cleaningProducts}) => ({
  products: cleaningProducts,
})

const mapDispatchToProps = dispatch => ({
  fetchProducts: () => dispatch(requestCleaningProducts()),
})

export default connect(mapStateToProps, mapDispatchToProps)(Cleaning)
