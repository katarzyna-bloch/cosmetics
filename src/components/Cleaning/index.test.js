import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Cleaning from '.'

it('should render correctly', () => {
  const wrapped = shallow(<Cleaning />)
  expect(toJson(wrapped)).toMatchSnapshot()
})
