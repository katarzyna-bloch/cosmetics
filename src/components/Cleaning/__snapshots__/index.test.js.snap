// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<h4>
  Oczyszczanie to podstawowy krok w dojściu do pięknej skóry
</h4>
`;
